scripts/release_issue_status/utils.py

```diff
@@ -181,7 +187,9 @@ def run_pipeline(issue_link, sdk_issue_object, pipeline_url):
     credentials = BasicAuthentication('', personal_access_token)
     run_parameters = models.RunPipelineParameters(**paramaters)
     client = PipelinesClient(base_url=organization_url, creds=credentials)
-    result = client.run_pipeline(project='internal',pipeline_id=2500,run_parameters=run_parameters)
+    # just debug
+    result = client.run_pipeline(project='internal', pipeline_id=2384, run_parameters=run_parameters)
```

Comment:
needn't change `pipeline_id` since we will merge `auto-release-debug` to `main` before this pr